# HG changeset patch
# User Sergey Mironov <grrwlf@gmail.com>
# Date 1404713104 -14400
#      Mon Jul 07 10:05:04 2014 +0400
# Node ID d318f3a24fe9964e218326d9610a3d6ac424d7b7
# Parent  9907088f9fa81773e124a551b1d99cdc7d11f9d1
Add 'role' data attribute.

Note, that 'role' attribute is a part of reach ARIA API described here:
http://www.w3.org/TR/wai-aria/

Among 'role', it defines lots of aria-* attributes

diff --git a/lib/ur/basis.urs b/lib/ur/basis.urs
--- a/lib/ur/basis.urs
+++ b/lib/ur/basis.urs
@@ -845,7 +845,7 @@
 con boxEvents = focusEvents ++ mouseEvents ++ keyEvents ++ resizeEvents ++ scrollEvents
 con tableEvents = focusEvents ++ mouseEvents ++ keyEvents
 
-con boxAttrs = [Data = data_attr, Id = id, Title = string] ++ boxEvents
+con boxAttrs = [Data = data_attr, Id = id, Title = string, Role = string] ++ boxEvents
 con tableAttrs = [Data = data_attr, Id = id, Title = string] ++ tableEvents
 
 val span : bodyTag boxAttrs
